workspaces/adr/.changeset/calm-mirrors-mate.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/search-backend-module-adr': minor
```

Comment:
@backstage/community-plugins-maintainers any guidance on whether search modules should reside within the main monorepo or with specific plugins here? eg. the explore search module still lives in the main repo: https://github.com/backstage/backstage/tree/master/plugins/search-backend-module-explore

Reply:
I think it's fine to be here!